clients/views.py

```diff
@@ -79,12 +76,10 @@ def get(self, *args, **kwargs):
 		return render(self.request, self.template_name, context=self.get_context_data(**kwargs))
 
 	def post(self, *args, **kwargs):
-		#import pdb; pdb.set_trace()
-		client_form = ClientForm(self.request.POST,user=self.request.user)	
-		#import pdb; pdb.set_trace()
+		client_form = ClientForm(self.request.POST,user=self.request.user )	
 		if client_form.is_valid() :
```

Comment:
use the power of django. don't nest your code as much as possible

https://stackoverflow.com/questions/19363527/my-code-nests-too-deep-is-there-a-better-way

you can do client_form.is_valid(exception=True)